src/routes/index.js

```diff
@@ -21,26 +21,28 @@ const Routes = () => {
   return (
     <Switch>
       <Route
-        path={'/vaccines'}
+        path={'/coronavirus-volunteer-map'}
         render={() => <Vaccines assets={filteredAssets} />}
       />
       <Route
-        path={'/timelines'}
+        path={'/coronavirus-timeline-tracker'}
         render={() => <MilestonesGraphContainer pins={filteredAssets} />}
       />
-      <Route path={'/charts'} render={() => <Charts pins={filteredAssets} />} />
+      <Route
+        path={'/covid-trial-vaccine-charts'}
+        render={() => <Charts pins={filteredAssets} />}
+      />
       <Route path={'/team'} render={() => <Team />} />{' '}
-      <Route path={'/charts'} render={() => <Charts pins={filteredAssets} />} />
       <Route path={'/faq'} render={() => <Iframe url={faqUrl} title='FAQ' />} />
       <Route path={'/contact'} render={() => <ContactUs />} />
       <Route
-        path={'/volunteer'}
+        path={'/volunteer-covid-trial'}
         render={() => (
           <Iframe url={howYouCanHelpUrl} title='How and Why to Volunteer' />
         )}
       />
       <Route
-        path={'/trialssummary'}
+        path={'/covid-trials-summary'}
```

Comment:
nice!

Reply:
I love this new routes names.

Could it all be nested inside `vaccines`? something like `/vaccines/coronavirus-timeline-tracker`

That to be able to match the new menu we are doing.

I just asking if it makes sense on SEO side, we can merge it like this, and then once the menu is done I can update the routes to work with that.